### DISCUSSION TITLE
I keep getting emails from unsubscribed discussions

### DISCUSSION THREAD

**Discussion by alan933** [community/community#3756] [Discussions]
This is literally spam! How do I stop getting github notifications emailed to me after unsubscribing????

**Comment by altela** [comment]
Hi,
Did you selected "Only receive account related emails, and those I subscribe to." ? in Settings > Email ?

![Screen Shot 2021-04-14 at 08 40 22](https://user-images.githubusercontent.com/68892527/114638062-5adca280-9cfd-11eb-951a-9140d4f26237.png)

**Comment by alan933** [comment]
I don't know where that is you are referring to.
Why can't I just click unsubscribe from topic like other sites do?

________________________________
